### DISCUSSION TITLE
[MacOS] Writing to text file doesn't work

### DISCUSSION THREAD

**Discussion by bstrother1** [unoplatform/uno#9128] [Q&A] [answered]
Not sure if I'm doing something wrong, this is a change with net6-macos, or what but doing:

https://stackoverflow.com/questions/48372705/xamarin-mac-os-x-save-and-retrieve-data-from-file

just produced PlatformNotSupported_Unix . Is this a bug or something that's not supported?

**Comment by jeromelaban** [comment, marked as answer]
Can you provide a more detailed stack trace?

**Reply by bstrother1** [reply]
nvm it was a different library crashing.
